nimbleclient/v1/fields.py

```diff
@@ -0,0 +1,1177 @@
+#
```

Comment:
I would merge this file with `field.py`

Reply:
I have kept them separate so that the *fields.py* is auto-generated completely (from SM XML files). The other file *field.py* is maintained only in github repo.